### PR TITLE
Added Logical And Double Overbar , Logical Or Double Overbar

### PR DESCRIPTION
**Pull request by satvikDesktop**
Logical or with Double Overbar - [Unicode Char Table](https://unicode-table.com/en/2A62/)
 Logical and with Double Overbar - [Unicode Char Table](https://unicode-table.com/en/2A5E/)

### CODE REVIEW

File: dictionary.txt [hunk 1]
```diff
@@ -907,3 +907,5 @@ _rp	₎
 \QED	∎	
 \male	♂	
 \female	♀	
+\logicalandbar  ⩞ 	
+\logicalorbar ⩢
```
Comment (DenverCoder1) [inline dictionary.txt:911]: I was able to find these LaTeX shortcuts in reference to these symbols. Perhaps these names could be used instead?

```suggestion
\doublebarvee	⩢	
\doublebarwedge	⩞	
\perspcorrespond	⩞	
```

**Comment by DenverCoder1** [comment]
There is no need to close a PR by the way, you can always make changes by making more commits on the same branch (in your case https://github.com/satvikDesktop/LaTeX-Gboard-Dictionary/tree/master)

**Comment by satvikDesktop** [comment]
> There is no need to close a PR by the way, you can always make changes by making more commits on the same branch (in your case https://github.com/satvikDesktop/LaTeX-Gboard-Dictionary/tree/master)

Oh , I 'm sorry to close the pull request . Anyways , should I rename the symbols and commit once again ?

**Comment by DenverCoder1** [comment]
Yeah, you can update them on your branch to fix it, or if you prefer a simpler way, you can click "commit suggestion" under my comment.

**Review by DenverCoder1 — APPROVED** [review]
Looks great! Thanks so much for contributing! :tada:

**Pull request merged**